x-pack/plugins/canvas/canvas_plugin_src/functions/common/compare.js

```diff
@@ -19,13 +23,16 @@ export const compare = () => ({
       aliases: ['_'],
       types: ['string'],
       default: 'eq',
-      help:
-        'The operator to use in the comparison: ' +
-        ' eq (equal), ne (not equal), lt (less than), gt (greater than), lte (less than equal), gte (greater than eq)',
+      help: i18n.translate('xpack.canvas.functions.compare.argsOpHelpText', {
+        defaultMessage:
+          'The operator to use in the comparison: eq (equal), ne (not equal), lt (less than), gt (greater than), lte (less than equal), gte (greater than eq)',
```

Comment:
I believe we can put all operators like eq, gt, lte to values. Only operators. Descriptions like (not equal), (less than) we do not need to put to values